tools/repro-build.sh

```diff
@@ -53,6 +53,7 @@ else
     VER=$(uname -r)
 fi
 
+ARCH=$(dpkg --print-architecture)
```

Comment:
Unrelated nit, but iirc breaking your "changlog" notes into two lines will cause problems for the tool that auto reads them. eg 

```
Changelog-Changed: Added architecture identifier to Ubuntu release
artifacts. ([https://github.com/ElementsProject/lightning/issues/6228])
```

should be 

```
Changelog-Changed: Added architecture identifier to Ubuntu release artifacts. ([https://github.com/ElementsProject/lightning/issues/6228])
```